src/org/loklak/api/search/SearchServlet.java

```diff
@@ -6,12 +6,12 @@
  *  modify it under the terms of the GNU Lesser General Public
  *  License as published by the Free Software Foundation; either
  *  version 2.1 of the License, or (at your option) any later version.
- *  
```

Comment:
Please revert this line, it's not part of the issue. If you want to do some house-keeping or cleaning, please create a new PR and Issue for it.